builtin/providers/vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -60,6 +61,7 @@ type virtualMachine struct {
 	timeZone             string
 	dnsSuffixes          []string
 	dnsServers           []string
+	bootableVmdkPath     string
```

Comment:
I would suggest to use a bool here instead - since essentially you are using this variable like a boolean further down in the file.